Mob/mob.gd

```diff
@@ -0,0 +1,11 @@
+extends CharacterBody2D
+
+@export var speed = 0
```

Comment:
Default speed is 0?

Reply:
we dont know what speed we want yet so it was set as 0 on default and we can decide together what speed we want it on.